### PR TITLE
chore: add unit tests for a component that has no tests

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
I did a fix for a component but this one didn't get any unit test
here are the tests for this component

### What issues does this PR fix or reference?

follow up of https://github.com/podman-desktop/podman-desktop/pull/10014

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+beforeEach(() => {
+  vi.clearAllMocks();
+  podsInfos.set(pods);
+});
+
+test('Expect switch of terminal clicking on the item', async () => {
+  render(KubernetesTerminalBrowser, { pod: podInfoUi });
+
+  // get the dropdown with id input-standard-My Pod"
```
Comment (feloy) [inline packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts]: it seems the comment is not up to date

File: packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+});
+
+test('Expect switch of terminal using keyboard switch with arrow keys', async () => {
+  // spyon terminal Service
+  const spyOnTerminalServiceSpy = vi.spyOn(terminalService, 'ensureTerminalExists');
+  podsInfos.set(pods);
+
+  render(KubernetesTerminalBrowser, { pod: podInfoUi });
+
+  // get the dropdown with id input-standard-My Pod"
```
Comment (feloy) [inline packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts]: same, not up to date

**Review by axel7083 — APPROVED** [review]

File: packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts (hunk 2 above)
Comment (benoitf) [reply packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts]: fixed

File: packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts (hunk 1 above)
Comment (benoitf) [reply packages/renderer/src/lib/pod/KubernetesTerminalBrowser.spec.ts]: fixed

**Review by feloy — APPROVED** [review]

**Pull request merged**